.github/workflows/tests.yml

```diff
@@ -1,10 +1,14 @@
 name: 'Tests'
 
 on:
-  push:
-    branches:
-      - main
   pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - '**/**.md'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
```

Comment:
Isn't this concurrency setting already scoped to the workflow?

Reply:
No, if we remove the workflow from that identifier then every other job from other workflows will get cancelled, you can see that [here](https://docs.github.com/en/actions/using-jobs/using-concurrency#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow)

Reply:
Good to know 👍